docs/cvl/methods.md

```diff
@@ -122,6 +123,48 @@ Wildcard entries may not have {ref}`envfree` or {ref}`optional`; their only
 purpose is {ref}`summarization <summaries>`.  Therefore, wildcard entries must
 have a summary.
 
```

Comment:
```suggestion
have a summary.

It is possible for a wildcard entry to overlap with an exact or wildcard entry; see {ref}`summary-resolution` for how summaries are resolved.

```

Reply:
Github won't let me leave a suggestion, but also include this (s/wildcard/exact) in the exact entry section.